tests/integration/test_author_api.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import six
+
+
+def test_author_main_path(app):
+    """This test checks only if the status cod and author profile are correct.
+    The functionality is provided by invenio-records-rest, which is not the
+    subject of these tests.
+    """
+    with app.test_client() as client:
+        assert client.get(
+            '/api/authors/983220').status_code == 200
+
+        request = json.loads(
+            client.get('/api/authors/983220').data)
+        print(request)
+        assert request['id'] == 983220
+
+
+def test_author_citations_endpoint(app):
+    """This test checks for the total count of objects and the availability
+    of required keys and their type.
+    """
+    with app.test_client() as client:
+        assert client.get(
+            '/api/authors/983220/citations/').status_code == 200
```

Comment:
Mh. Just noticed this:
```
% http --headers http://inspire.jacenkow.com:5000/api/authors/984446/publications/
HTTP/1.1 200 OK

% http --headers http://inspire.jacenkow.com:5000/api/authors/984446/publications
HTTP/1.1 301 MOVED PERMANENTLY
```
(Note the missing `/` at the end of the URL). I think we want URLs without a trailing slash?